.github/scripts/build-default-workloads.py

```diff
@@ -13,8 +13,9 @@ def build_default_workloads():
         # avoid logging excessive amounts to prevent GH-A masking secrets (which slows down log output)
         with settings(warn_only=True):
             rc = run("marshal -v build br-base.json &> br-base.full.log").return_code
+            print(f"marshal exit code was '{rc}'")
+            run("cat br-base.full.log")
```

Comment:
Is this change supposed to be included? In that case, the comment above it is no longer correct.

Reply:
No, I intended to revert that after fixing it and also addressing https://github.com/firesim/FireMarshal/issues/233 Thanks for pointing out.  I'll revert now.

Reply:
I actually decided to wait and see if there is more feedback I need to apply before rerunning CI.

Reply:
I reverted this in my latest push.  @sagark , can you take another look through and approve?